packages/site/src/utils/theme.ts

```diff
@@ -0,0 +1,42 @@
+import { Theme } from "@jobber/components";
+
+const FALLBACK_THEME: Theme = "light";
```

Comment:
Are these helpers that we should be shipping?! Have we just improved the design system like 3 times as a result of this PR??

Reply:
Fair!